### PR TITLE
add layout_catalog task solution

### PR DESCRIPTION
**Pull request by opiquem**
[DEMO LINK](https://maksym-bobryk.github.io/layout_catalog/)
[TEST REPORT LINK](https://maksym-bobryk.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    height: 40px;
+
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+
+    background: $blue-color;
+    border: 1px solid $blue-color;
```
Comment (amashutikov) [inline src/styles/blocks/card.scss:80]: In previous task 'card' we should do transparent border and change its color on hover.

**Review by julshvets — CHANGES_REQUESTED** [review]
Great job, but I have some suggestions and recommendations.

File: src/styles/blocks/stars.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  &--1 :nth-child(-n + 1),
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
+    background-image: url("../images/star-active.svg");
+  }
+
```
Comment (amashutikov) [inline src/styles/blocks/stars.scss:22]: Checklist item 12 we should try to use @for loop for this.

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+@import "./blocks/page.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/catalog.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
 body {
   margin: 0;
 }
```
Comment (amashutikov) [inline src/styles/main.scss:13]: I think we should move this to reset.scss

**Review by amashutikov — APPROVED** [review]
Great job, added small improvments.

File: src/index.html [hunk 4]
```diff
@@ -1,12 +1,464 @@
...
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="moyo"
+        class="logo header__logo"
+        >
```
Comment (Anna-Tkachenko) [inline src/index.html:30]: Check correct indents for attribtues

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,464 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="."
+              class="nav__link nav__link--is-active is-active"
+            >
+              APPLE
```
Comment (Anna-Tkachenko) [inline src/index.html:40]: Let's transform text with css property instead of do it directly in HTML

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,464 @@
...
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="." class="nav__link">SAMSUNG</a>
```
Comment (Anna-Tkachenko) [inline src/index.html:45]: Add corresponding href according to the content for each link
```suggestion
            <a href="#samsung" class="nav__link">SAMSUNG</a>
```

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,464 @@
...
+          <li class="nav__item">
+            <a href="." class="nav__link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <div class="catalog__container">
```
Comment (Anna-Tkachenko) [inline src/index.html:83]: Do you need 2 wrappers for catalog? Let's get rid one of them

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Well Done!
Just check some suggestions for HTML structure

File: src/styles/blocks/catalog.scss [hunk 8]
```diff
@@ -0,0 +1,34 @@
+.catalog {
+  box-sizing: border-box;
+  margin: 0 auto;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/catalog.scss:3]: You can't set margins to BEM block

**Review by Anna-Tkachenko — APPROVED** [review]
Good job! ⭐️

**Pull request closed without merging**